src/nxdoc/nuxeo-server/additional-services/conversion.md

```diff
@@ -387,6 +388,16 @@ To contribute a new converter, you have to contribute a class that implement the
 </extension>
 ```
 
+Here is an example of conversion bypass on the any2pdf converter, the input and output is pdf and we decided we don't need to process the file:
```

Comment:
For "and we decided we don't need to process the file" I'd say -> "in which case we must not process the file", as otherwise there's a bug.